src/api/queries/blockchainQueries.js

```diff
@@ -1,17 +1,18 @@
 ApiClient_.prototype.fetchActiveAddresses = function (slug, from, to) {
   var query = {
     'query': '{\
-       dailyActiveAddresses(slug: "' + slug + '",\
-                            from: "' + toUTC_(from) + '",\
-                            to: "' + toUTC_(to) + '",\
-                            interval: "1d") {\
-         activeAddresses\
-         datetime\
-       }\
-     }'
+       getMetric(metric: "daily_active_addresses") {\
```

Comment:
Why is needed? You can just remove the old code, new function will be generated from getMetric.